src/pages/registration/resource_type_tab/sub_type_forms/research_data_types/RelatedResultItem.tsx

```diff
@@ -38,22 +38,14 @@ export const RelatedResultItem = ({
   const isConfirmedDocument = document.type === 'ConfirmedDocument';
 
   return (
-    <ListItem
-      sx={{
-        display: 'flex',
-        gap: '1rem',
-        bgcolor: 'white',
-        borderRadius: '0.25rem',
-        border: '1px solid lightgray',
-        my: '0.25rem',
-        minHeight: '5rem',
-      }}>
+    <ListItem disablePadding sx={{ mb: '0.5rem' }}>
       <Box
         sx={{
           display: 'grid',
           gridTemplateColumns: '1fr 1fr',
           alignItems: 'center',
           minWidth: '4rem',
+          mr: '0.5rem',
```

Comment:
kunne hatt gap på fleksen i stedet, sånn at det ble space bare mellom elementene, og slipper at det siste elementet også får en ekstra margin (om det ikke var ønskelig da 😅)

```suggestion
    <ListItem disablePadding sx={{ mb: '0.5rem', gap: '0.5rem' }}>
      <Box
        sx={{
          display: 'grid',
          gridTemplateColumns: '1fr 1fr',
          alignItems: 'center',
          minWidth: '4rem'
```